README.md

```diff
@@ -31,3 +31,34 @@ Contributors
 Contributing
 ------------
 Please read more about [Github Flow](https://guides.github.com/introduction/flow/).
+
+Getting started
+---------------
+
+In order to customize docker skeleton:
```

Comment:
imho better option would be step by step instructions how to use it within existing project